packages/worker/src/worker-options.ts

```diff
@@ -195,10 +195,31 @@ export interface WorkerOptions {
   /**
    * Maximum number of Workflow tasks to execute concurrently.
    * Adjust this to improve Worker resource consumption.
+   * Can't be lower than 2 if `maxCachedWorkflows` is non-zero.
    * @default 100
```

Comment:
We should lower this to avoid a large backlog and possibly timeouts

Reply:
Now, regarding `maxConcurrentActivityTaskExecutions` and `maxConcurrentLocalActivityTaskExecutions`, I would lower both default values to something in the range of 20-40, but we should really encourage people to self configure these based on their own workload.